test/unit/backend/server.spec.js

```diff
@@ -27,4 +27,16 @@ describe('Server', () => {
         });
     });
   });
+
+  describe('/api/movies', () => {
+    it('should GET all the movies', (done) => {
+      chai.request(server)
+        .get('/api/movies')
+        .end((err, res) => {
+          res.should.have.status(200);
+          res.body.should.be.a('array');
+          done();
+        });
+    });
+  });
 });
```

Comment:
Although not very good to merge without as close as 100% test coverage possible, I think it's best for us to merge this now (as it's actually getting quite big) and create a task to deal with that later.